cardano-cli/src/Cardano/CLI/Shelley/Parsers.hs

```diff
@@ -1066,6 +1067,14 @@ pMinFeeConstantFactor =
     <> Opt.help "The constant factor for the minimum fee calculation."
     )
 
+pMinUTxOValue :: Parser (Maybe Natural)
+pMinUTxOValue =
+  Opt.option Opt.auto
```

Comment:
I think this needs to use `Opt.optional` otherwise, we're `Opt.auto` is using `read` at type `Maybe Natural` and of course that's the wrong text format "Just 3" rather than "3".

Reply:
Looks like I need to change the other parsers too. Will do so quickly.